src/lazy/expanded/macro_evaluator.rs

```diff
@@ -757,8 +1069,7 @@ mod tests {
     }
 
     // TODO: macros_inside_a_tdl_sexp()
-    // This requires an implementation of TDL's `(make_sexp)` or `(quote)`. Without these,
-    // a sexp is always considered a TDL macro invocation.
+    // This requires an implementation of TDL's `(make_sexp)`.
```

Comment:
🗺️ You actually can't use `quote` to make an s-expression if you want it to have nested macro invocations. `quote` will cause the nested invocations to be considered s-expressions too.